.travis.yml

```diff
@@ -39,9 +59,15 @@ before_install:
 
 install:
     - ./autogen.sh
+    # Build rdma-core only in linux as OS X doesn't have verbs support
+    - if [[ "$TRAVIS_OS_NAME" == "linux" ]] ; then
+        RDMA_CORE_BRANCH=v20 ;
+        git clone --depth 1 -b $RDMA_CORE_BRANCH https://github.com/linux-rdma/rdma-core.git && cd rdma-core && bash build.sh && cd - ;
```

Comment:
That is what the -b $RMDA_CORE_BRANCH is supposed to resolve. I picked the v20 tag as the most recent tag they had available.

Reply:
The v20 tag matches up with the rdma-core-20 release. It would be simple enough to change the tag to something else. I can try with 20.1. Do either of you have a preference?